packages/context-menu/theme/material/vaadin-context-menu.js

```diff
@@ -1,4 +1,5 @@
 import './vaadin-context-menu-styles.js';
 import '@vaadin/item/theme/material/vaadin-item.js';
+import '@vaadin/vaadin-overlay/theme/material/vaadin-overlay.js';
```

Comment:
nit: let's sort import alphabetically (place this one after `@vaadin/list-box` import below).

Reply:
Done.